middleware/logger_test.go

```diff
@@ -106,92 +112,10 @@ func TestLoggerHeaders(t *testing.T) {
 	h(c)
 
 	assert.Contains(t, buf.String(), "AAA-CUSTOM-VALUE")
```

Comment:
Do you want to extend the test to cover all tags? Just do a positive test on tag e.g. `Contains`.